tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,
-    py27-pyramid{12,13,14,15,16},
```

Comment:
Instead of removing testing against old versions of Pyramid, we should update this to newer versions of Pyramid to test against instead, and add `py37-pyramid{20}` when the time comes.

Reply:
do not remove testing against old versions of pyramid

Reply:
Should be fixed in 494bdfdf39ee244fd242dc468e39c335006b2778. Please verify.